deploy/helm_charts/envs/bard_staging.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DNS
+ip:
+domain: staging.bard.datacommons.org
+
+# Helm config
+project: "datcom-bard-staging"
+cluster_prefix: "website"
+
+namespace:
+  name: "website"
+
+website:
+  flaskEnv: custom
+  replicas: 5
+  redis:
+    enabled: true
+    configFile: |
+      {
+        "us-central1": {
+          "host": "10.50.6.211",
+          "port": "6379"
+        }
+      }
+  nodePool: "default-pool"
+
+serviceAccount:
+  name: website-ksa
+
+ingress:
+  enabled: true
+  internal: true
+
+nl:
+  enabled: true
+  memory: "10G"
+  workers: 3
+  env:
+    default_indexes:
+      - base_uae_mem
+    enabled_indexes:
+      - medium_ft
+      - sdg_ft
+      - undata_ft
+      - base_uae_mem
+    vertex_ai_models:
+      uae-large-v1-model:
+        project_id: datcom-nl
+        location: us-central1
+        prediction_endpoint_id: "1309973546495115264"
+
+nodejs:
+  enabled: true
+  replicas: 5
+  nodePool: "default-pool"
+
+serviceGroups:
+  recon: null
+  svg:
+    replicas: 3
+    nodePool: "default-pool"
+  node:
+    replicas: 5
+    nodePool: "default-pool"
+  observation:
+    replicas: 3
+    nodePool: "default-pool"
+  default:
+    replicas: 3
+    nodePool: "default-pool"
+
+svg:
+  blocklistFile: ["dc/g/Uncategorized", "oecd/g/OECD"]
+
+cronTesting:
```

Comment:
we probably don't need cronTesting for staging instance?

Reply:
removed